### DISCUSSION TITLE
Building a themeable Design system

### DISCUSSION THREAD

**Discussion by yaldram** [joe-bell/cva#126] [Show and tell]
- I wrote a blog tutorial series on building a theme able design system with React, Typescript, Scss and class-variance-authority.
- This design system is an improvement over my previous one, a chakra ui clone which was build using React, Typescript, styled-components and styled-system.
- This design system also has build in dark mode.
- I would encourage you to play around with the [deployed storybook](http://yaldram-react-scss-chakra-ui.s3-website.ap-south-1.amazonaws.com/?path=/story/atoms-forms-button--default&args=colorScheme:red).
- You can find the tutorial series on [dev.to](https://dev.to/yaldram/build-a-design-system-with-dark-mode-using-react-typescript-scss-cva-and-vite-setup-5778).
- All the code for this tutorial series is on [github](https://github.com/yaldram/react-scss-chakra-ui).

**Comment by joe-bell** [comment]
Nice work! 🔥

**Comment by AdsonCicilioti** [comment]
Inspiration! 🤩

**Comment by yaldram** [comment]
Hello Community, I published another tutorial series on building theme able design systems, in this design system we are using CSS variables for theming please check this - https://github.com/joe-bell/cva/discussions/156
